### DISCUSSION TITLE
PnP-GroupMembers

### DISCUSSION THREAD

**Discussion by mmgwo** [pnp/powershell#3914] [General]
![Screenshot 2024-04-25 134459](https://github.com/pnp/powershell/assets/168090921/5bca3fc4-53c4-462c-b053-9b3ab1d42c4a)
I keep receiving this error and have been trying to fix it for a while now. Can anyone support here? Any help would be greatly appreciated!

**Comment by ildjarnisdead** [comment]
It's Get-PnPGroupMember (without s), and the parameter is -Group, not -Identity.
https://pnp.github.io/powershell/cmdlets/Get-PnPGroupMember.html
